### PR TITLE
svgwrite: intial release 0.1

### PR DESCRIPTION
**Pull request by dvd0101**
Specify library name and version:  **svgwrite/0.1**

- [x ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/svgwrite/0.1.0/conandata.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+sources:
+  "0.1":
+    url: "https://gitlab.com/dvd0101/svgwrite/-/archive/master/svgwrite-master.tar.gz"
```
Comment (uilianries) [inline recipes/svgwrite/0.1.0/conandata.yml:3]: master is not a good reference, it changes for each new release. You have two options:
- Create a tag and refer it on this url (best choice)
- Use the date of that commit as version and set the commit as package for downloading

https://github.com/conan-io/conan-center-index/wiki/FAQ#what-version-should-packages-use-for-libraries-without-official-releases

File: recipes/svgwrite/0.1.0/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class SvgwriteConan(ConanFile):
+    name = "svgwrite"
+    version = "0.1"
```
Comment (danimtb) [inline recipes/svgwrite/0.1.0/conanfile.py:7]: Should this be 0.1.0?

File: recipes/svgwrite/0.1.0/conandata.yml (hunk 1 above)
Comment (dvd0101) [reply recipes/svgwrite/0.1.0/conandata.yml]: I opted for a tag! TY

File: recipes/svgwrite/0.1.0/conanfile.py (hunk 2 above)
Comment (dvd0101) [reply recipes/svgwrite/0.1.0/conanfile.py]: fixed! The control performed by the bot,  which found the inconsistency between the versions, could it be added to the hooks?

**Comment by dvd0101** [comment]
> @dvd0101 Consider [dvd0101#1](https://github.com/dvd0101/conan-center-index/pull/1)

Merged! Is `N versions => 1 recipe` your preferred method for a conan recipe?

**Comment by uilianries** [comment]
> 
> 
> > @dvd0101 Consider [dvd0101#1](https://github.com/dvd0101/conan-center-index/pull/1)
> 
> Merged! Is `N versions => 1 recipe` your preferred method for a conan recipe?

Yes, if possible. It keeps the things easier to be maintained.

**Pull request merged**